x/eibc/client/cli/query_command_orders.go

```diff
@@ -68,11 +93,62 @@ func CmdListDemandOrdersByStatus() *cobra.Command {
 				return fmt.Errorf("failed to fetch demand orders: %w", err)
 			}
 
-			return clientCtx.PrintProto(res)
+			out, err := cmd.Flags().GetString("output")
+			if err == nil && out == "json" {
+				return clientCtx.PrintProto(res)
+			}
+
+			count := 0
+
+			for _, o := range res.DemandOrders {
+				count++
+
+				fmt.Printf(`
+-id:		%s
+  recipient:	%s
+  price:	%s
+  fee:		%s
+  rollapp_id:	%s
+  status:	%s
+  packet_key:	%s
+  fulfiller:	%s
+`,
+					o.Id, o.Recipient, parseAndFormat(o.Price.String()), parseAndFormat(o.Fee.String()), o.RollappId,
+					o.TrackingPacketStatus.String(), strings.TrimSpace(o.TrackingPacketKey), o.FulfillerAddress)
+			}
+
+			fmt.Printf("\ncount: %d; ts: %s\n", count, time.Now().String())
```

Comment:
It's still possible to use `-o json` right? to get json output?

Reply:
also: len(res.DEmandOrders) instead of the count?

Reply:
yes, `-o json` works still